src/data-access/MwApiLexemeCreator.ts

```diff
@@ -1,11 +1,12 @@
-import { MwApi } from '@/@types/mediawiki';
+import { MwApi, MwUtilGetUrl } from '@/@types/mediawiki';
 import LexemeCreator from '@/data-access/LexemeCreator';
 import { SubmitError } from '@/store/RootState';
 
 type WbEditEntityResponse = {
 	entity: {
 		id: string;
 	};
+	tempuserredirect: string;
```

Comment:
```suggestion
	tempuserredirect?: string|null;
```

If no temp user was created, the redirect won’t be present, and if it was created, it might still be `null` if no redirect is needed.

Reply:
done!